### ISSUE TITLE
Incremental map rendering

### ISSUE THREAD

**Issue by wonder-sk** [lutraconsulting/input#157]
We can try to enable incremental map rendering of qgis quick canvas (`incrementalRendering: true`). Need to test thoroughly if it is working well (tiled layers, multiple layers, ...) and if the UX is not better, we can leave it until things improve in qgis quick.

**Comment by PeterPetrik** [comment]
https://github.com/lutraconsulting/input/blob/8bdbf81e9064628b2d8ae873a3983afe907d0cb7/qgsquick/qgsquickmapcanvasmap.h#L120

looks like we just need to set it on?

**Comment by wonder-sk** [comment]
Yes but it needs good testing, the UX could actually get worse in some cases. Right now if you move map, the old map image is still there until map gets fully refreshed. The incremental rendering may erase the previous map image soon after refresh, and user may loose context (for a few seconds) before the map is refreshed... But maybe it's not happening anymore...
